js/utils/templateHelpers.js

```diff
@@ -50,16 +50,14 @@ export function parseEmojis(text, className = '', attrs = {}) {
 
 /**
  * If the average is a number, show the last 2 digits and trim any trailing zeroes.
- * Don't show the count if the count is invalid, x (0) would be inaccurate and confusing.
- * If the average is invalid or 0, don't show anything.
  * @param {number} average - the average rating
  * @param {number} count - the number of ratings
  */
 export function formatRating(average, count) {
   const avIsNum = typeof average === 'number';
-  const ratingAverage = avIsNum ? average.toFixed(1) : '';
-  const ratingCount = typeof count === 'number' ? ` (${count})` : '';
-  return avIsNum && average > 0 ? `${parseEmojis('⭐')} ${ratingAverage}${ratingCount}` : '';
+  const ratingAverage = avIsNum ? average.toFixed(1) : 0;
```

Comment:
I understand to show 0 if the counts are coming back as 0, but do we really want to show 0 if the counts aren't provided? Isn't that really misleading, and potentially shocking to a vendor, who may have 100 reviews with 5 stars, but for some reason a search provider doesn't provide that field and we end up representing the listing as having no reviews?